### DISCUSSION TITLE
I have a Problem

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6455] [General]
well i am getting the m3u8 file to an external player , once there is no stream , i have no way to put anything instead of a black or stream not found on the player  , because the m3u8 file is been deleted once the stream is over , I need a way to keep m3u8 file with an image to say that the stream is not active , and once the stream start , this m3u8 file is replaced with the live m3u8 , no idea how to make this happen , i hope AMS team have a solution  ,

**Comment by Mohit-3196** [comment]
Hi @brixxnatt,
If you use the play.html, [this solution](https://github.com/orgs/ant-media/discussions/4877) can work for you.

But since you are using an external player, you should consider adding an overlay/logo on the client Side.
Overlay the logo on the client player side using some basic CSS and images you can have a fake overlay on the video player.

Thank you
